helix-rest/src/main/java/org/apache/helix/rest/server/resources/helix/InstancesAccessor.java

```diff
@@ -158,7 +159,9 @@ public Response instancesOperations(@PathParam("clusterId") String clusterId,
       @QueryParam("continueOnFailures") boolean continueOnFailures,
       @QueryParam("skipZKRead") boolean skipZKRead,
       @QueryParam("skipHealthCheckCategories") String skipHealthCheckCategories,
-      @DefaultValue("false") @QueryParam("random") boolean random, String content) {
+      @DefaultValue("false") @QueryParam("random") boolean random,
+      @DefaultValue("false") @QueryParam("skipCustomChecksIfNoLiveness") boolean skipCustomChecksIfNoLiveness,
```

Comment:
I prefer not to have them as query param. The URL will be very very long and not easy to propose. 

Let's try to make them part of payloads. Payloads are more flexible and not have backward compatibility issues.